CMakeLists.txt

```diff
@@ -42,7 +44,10 @@ target_link_libraries(violet PRIVATE LibJuice::LibJuiceStatic)
 
 install(TARGETS violet RUNTIME DESTINATION bin)
 
-target_compile_options(violet PRIVATE -Wall -Wextra)
+target_compile_options(violet PRIVATE -Wall)
```

Comment:
Do `-Wall` work for MSVC? Is it translated by the CMake generator?

Reply:
Well it didn't error at least